src/main/java/com/softserve/academy/spaced/repetition/dto/Request.java

```diff
@@ -0,0 +1,4 @@
+package com.softserve.academy.spaced.repetition.dto;
+
+public interface Request {
```

Comment:
Please add javadoc explaining why do you add this class with so common name.

Reply:
@antonbakalets , it's wrong?)

Reply:
I saw it here:
https://lmonkiewicz.com/programming/get-noticed-2017/spring-boot-rest-request-validation/

Reply:
It is not wrong. The problem is that when you return to this code in 2-3 months you will not remember, it needs a comment, preferably a javadoc.